pkg/compiler/backend/simpleimg.go

```diff
@@ -37,24 +39,51 @@ func (*SimpleImg) BuildImage(opts compiler.CompilerBackendOptions) error {
 	name := opts.ImageName
 	path := opts.SourcePath
 	context := opts.Context
+	writer := NewBackendWriter(!config.LuetCfg.GetGeneral().ShowBuildOutput)
 
 	if context == "" {
 		context = "."
 	}
 	dockerfileName := opts.DockerFileName
 
 	buildarg := []string{"build", "-f", dockerfileName, "-t", name, context}
-	Spinner(22)
-	defer SpinnerStop()
+
+	if config.LuetCfg.GetGeneral().ShowBuildOutput {
+		// We have realtime output from command. Quiet spinner.
```

Comment:
No need to quiet it, your change is actually dropping the spinner to start

Reply:
Not, i check this, the spinner on simpleimg was already active from compiler code. It was wrong there or unneeded.

Reply:
drop it from the compiler then, no need to handle the disabling here

Reply:
It's needed for the other phases. Unpack, etc.

Reply:
It's more verbose add it in every phase... downloading, etc.

Reply:
by searching `Spinner` on `compiler.go` it reveals to be used only in two places, so I still believe it's better that the backend decides when an operation would take time or not. It's _more_ confusing to just handle the spinner stop here, because it assumes something else started it before

Reply:
So do you want to add the if on `compiler.go` code?

Reply:
In this case, move the if there means that for tagging and other phases will not enable the spinner. I don't agree with this but as you prefer. Let me know.

Reply:
my bad I wasn't clear: I think in the compiler at that point we shouldn't handle spinner start/stop anymore

Reply:
so, how do you want to enable/disable the spinner for long tasks? It's not clear to me.

Reply:
long tasks are coming from the backend only, so starting and stop of spinner should only happen there.

Reply:
ping @geaaru shall I look over it?